deepinv/loss/regularisers.py

```diff
@@ -87,7 +98,7 @@ def forward(self, y, x):
                 v.detach_()
                 u.detach_()
 
-        return z.view(-1).sqrt()
+        return self.weight * z.view(-1).sqrt()
```

Comment:
I would not do this: this functions gives the spectral norm of the Jacobian, so if we multiply the max eigenvalue by `self.weight`, it is confusing. I would rather add the weight somewhere else. If we do not find anything clearer, then I would move the compute function inside a `def compute(self, ...)` method (or any other name) returning the proper computation, and the forward would just call `self.compute(...)` and multiply the output with `self.weight`. What do you think is best?

Reply:
good point. Maybe we can add the weights somewhere in the train function, and avoid adding them to the losses one by one.